core/src/main/java/com/linecorp/armeria/client/retry/RetryingRpcClient.java

```diff
@@ -124,9 +167,13 @@ private void doExecute0(ClientRequestContext ctx, RpcRequest req,
         final RpcResponse res = executeWithFallback(unwrap(), derivedCtx,
                                                     (context, cause) -> RpcResponse.ofFailure(cause));
 
+        final RetryConfig<RpcResponse> retryConfig = mapping().get(ctx, req);
+        final RetryRuleWithContent<RpcResponse> retryRule =
+                retryConfig.needsContentInRule() ?
```

Comment:
Should we leave a warning log if this is not `true`?

Reply:
Done